content/livesync/models.textile

```diff
@@ -193,8 +270,7 @@ try {
 <p>Be aware that updating the model state in your database happens entirely through your backend endpoint.</p>
```

Comment:
I think in the section above this, we need to include a code snippet that shows how the merge function works with an optimistic update vs a confirmed update.